README.md

```diff
@@ -6,18 +6,30 @@
 The Jaeger Operator is an implementation of a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
 
 ## Getting started
-
-To install the operator, run:
+1) Create the namespace: 
 ```
 kubectl create namespace observability
+```
+2) Load the CRD based on your version:
+- Kubernetes 1.12+:
+```
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
+```
+- Kubernetes 1.11-: 
+```
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd_ocp311.yaml
```

Comment:
Could you change the CRD filename to end `k8s1.11.yaml`? as listed under the k8s 1.11 header